### ISSUE TITLE
Confusion about Camel Contexts page and missing it in the Camel tree

### ISSUE THREAD

**Issue by jsolovjo** [hawtio/hawtio-next#319]
I noticed the following thing - when I open Camel I can see Camel Contexts page even though there is no Camel Contexts in Camel tree anymore. After I click on SampleCamel I can't find CamelContexts anymore but if I put directly `/contexts` into URL bar, I can see it again

![image](https://github.com/hawtio/hawtio-next/assets/49488040/39cbf7c1-f3bc-4ea9-ac06-6f61262b1cf3)

Is it somehow expected behaviour? If it was decided to get rid of CamelContexts meaning that only one Camel Context should be deployed, so perhaps it's better to remove Camel Contexts page too to avoid any confusion

What do you think?

**Comment by tadayosi** [comment]
@jsolovjo Yes, it's the expected behaviour. Paul and I discussed it when he was implementing #238, and we decided to make the Contexts view as default for Camel plugin when no node is selected. So, as the entrance view, it should list all the Camel contexts loaded (but it should be single anyway...).

What it means to get rid of CamelContexts node from the tree is that the node is static and never changes so it just comsumes the space in the left side tree without providing extra values. We just hide the node from rendering in the tree view and nothing more.
